lib/utils/use-ghost-user.js

```diff
@@ -16,21 +16,51 @@ module.exports = function useGhostUser(contentDir) {
         ghostuid = execa.shellSync('id -u ghost').stdout;
         ghostgid = execa.shellSync('id -g ghost').stdout;
     } catch (e) {
+        // CASE: the ghost user doesn't exist, hence can't be used
         if (!e.message.match(/no such user/i)) {
             throw new errors.ProcessError(e);
         }
 
-        return false;
+        return false
     }
 
     ghostuid = parseInt(ghostuid);
     ghostgid = parseInt(ghostgid);
 
+    return {
+        uid: ghostuid,
+        gid: ghostgid
+    };
+}
+
+const shouldUseGhostUser = function shouldUseGhostUser(contentDir) {
+    let ghostUser;
+
+    if (os.platform() !== 'linux') {
+        return false;
+    }
+
+    // get the ghost uid and gid
+    try {
```

Comment:
since this is just catching and re-throwing the error, we can just remove the try/catch block altogether.